internal/provider/nordvpn/api/location.go

```diff
@@ -0,0 +1,31 @@
+package api
+
+type location struct {
+	Country struct {
+		City struct {
+			Name string
+		}
+		Code string
+		Name string
+	}
+}
+
+func (s *server) country() string {
+	if s.Locations == nil || len(s.Locations) == 0 {
```

Comment:
nil and empty slice are both length 0 in Go

```suggestion
	if len(s.Locations) == 0 {
```